lib/src/main/java/org/team1540/rooster/drive/pipeline/CTREOutput.java

```diff
@@ -0,0 +1,118 @@
+package org.team1540.rooster.drive.pipeline;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.IMotorController;
+import java.util.Objects;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * {@link Output} to pass drive commands to Talon SRX and Victor SPX motor controllers. For output
```

Comment:
Wait so when are we using our wrappers and when are we not?

Reply:
Since I'm using none of the wrapper functions I thought it better to use `IMotorController`. (`ChickenController` extends that.)

Reply:
We should probably refactor some other stuff then, low priority